chapters/request_size/request_size.rst

```diff
@@ -0,0 +1,99 @@
+=================
+Request Size
+=================
+
+``request_size`` is the parameter that we could set for the Flow.
+It defines the number of ``Documents`` a single ``Request`` will contain,
+so ``Peas`` will receive requests containing this amount of ``Documents`` according to different
+scheduling policies.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Request Size
+------------
+Jina defines the ``request_size`` as the parameter at client side. By setting the ``request_size`` in Flow's API
+we could define the number of ``Documents`` contained in every ``Request``.
+
+Batch Size
+------------
+Batch size, commonly used in machine learning, often refers to the number of data that we feed into
+the model in one iteration.
+
+In Jina, ``batch_size`` is used at ``Driver`` and ``Executor`` level. Where
```

Comment:
In Jina batch size is used by drivers and Encoders to guarantee that the Executor processes data in pieces of a specific size

Reply:
Only Encoders use batch size?

Reply:
Done

Reply:
no sorry Executors, Indexers also do

Reply:
so better say Executors sorry

Reply:
Got it. ✅